### PR TITLE
add cli tx, query

### PR DESCRIPTION
**Pull request by dongsam**
WIP 

`./liquidityd tx liquidity create-liquidity-pool 1 100reservecoin1,100reservecoin2 --from node0  --home ./output/node0/liquidityd/`

when any query or tx 

`Error: {"codespace":"sdk","code":6,"message":"unknown query path"}`

the `unknown query path` err occurred and needs to be resolved, 
I spent hours but couldn't find it

### CODE REVIEW

**Comment by fdymylja** [comment]
Add the query types here: https://github.com/b-harvest/liquidity/blob/develop/x/liquidity/keeper/querier.go

File: x/liquidity/client/rest/query.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ func queryLiquidityHandlerFn(cliCtx client.Context) http.HandlerFunc {
 		vars := mux.Vars(r)
 		strPoolId := vars[RestPoolId]
 
-		poolID, ok := rest.ParseUint64OrReturnBadRequest(w, strPoolId)
+		poolId, ok := rest.ParseUint64OrReturnBadRequest(w, strPoolId)
```
Comment (jgimeno) [inline x/liquidity/client/rest/query.go:26]: poolID I think is what the linter normally suggest.

**Comment by dongsam** [comment]
Thanks for your help! Fortunately, I found a bug, which appears to have been returned from the sdk because `"unknown query path: %s"` instead of `"unknown query path: %s"`, so I tried to resolve it while trying to resolve the issue with rebase `0.40.0-rc4` to find the cause, probably incorrectly registered in `root.go`

File: x/liquidity/client/rest/query.go (hunk 1 above)
Comment (dongsam) [reply x/liquidity/client/rest/query.go:29]: Thank you for your opinion. I was originally using it as a poolID, but the snake_case of pool_id was automatically converted into a camel case of poolId, so I couldn't find a way to try it, so I'm using it as a temporary measure. In also latest version of cosmos-sdk, [proposal_id](https://github.com/cosmos/cosmos-sdk/blob/d55c1a26657a0af937fa2273b38dcfa1bb3cff9f/proto/cosmos/gov/v1beta1/gov.proto#L49) has been automatically converted to [ProposalId](https://github.com/cosmos/cosmos-sdk/blob/d55c1a26657a0af937fa2273b38dcfa1bb3cff9f/x/gov/types/gov.pb.go#L166).

**Pull request merged**